core/src/test/java/org/wso2/testgrid/core/command/GenerateTestPlanCommandTest.java

```diff
@@ -116,7 +117,8 @@ public void testExecute(String jobConfigFile, String workingDir) throws Exceptio
         InfrastructureParameter param = new InfrastructureParameter("ubuntu_16.04", DefaultInfrastructureTypes
                 .OPERATING_SYSTEM, "{}", true);
         InfrastructureCombination comb1 = new InfrastructureCombination(Collections.singleton(param));
```

Comment:
Shall we change the 'comb1' to a more meaningful name as well?

Reply:
Fixed with e6479361dd8952f6b1efed92fdf164956691cf0e